funnel/models/moderation.py

```diff
@@ -0,0 +1,56 @@
+from baseframe import __
+from coaster.utils import LabeledEnum
+
+from . import Comment, NoIdMixin, User, UuidMixin, db
+
+__all__ = ['MODERATOR_REPORT_TYPE', 'CommentModeratorReport']
+
+
+class MODERATOR_REPORT_TYPE(LabeledEnum):  # NOQA: N801
+    OK = (0, 'ok', __("OK"))
+    SPAM = (1, 'spam', __("Spam"))
+
+
+class CommentModeratorReport(UuidMixin, NoIdMixin, db.Model):
```

Comment:
Using a compound primary key and adding a unique UUID column looks like a design error. One of them is not required.

Reply:
Using UUID field as primary key.